Sources/Client/Model/Channel/ChannelsQuery.swift

```diff
@@ -45,6 +51,7 @@ public struct ChannelsQuery: Encodable {
                 sort: [Sorting] = [],
                 pagination: Pagination = [.channelsPageSize],
                 messagesLimit: Pagination = [.messagesPageSize],
+                memberLimit: Int = 100,
```

Comment:
needs to be added to the documentation + exposed up to the chain (rx extension etc.)